### PR TITLE
adding performance strata detailed info

### PR DESCRIPTION
**Pull request by jiahuang**
Does the following:
* Adds a script to look through quality measurement pdfs in https://qpp.cms.gov/docs/QPP_quality_measure_specifications.zip and extract out all the details about each performance strata
* adds a `measures/quality-measures-additional-info.json` that has shortened names of each performance strata along with algorithms for computing multi-performance measures
* updates the readme to reflect the measurements.json file that merges data from the above two datapoints
* bumps up version number

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,9 +44,20 @@ var benchmarksSchema = qppMeasuresData.getBenchmarksSchema();
 
 ### Generating measures-data.json
 To regenerate the `measures-data.json` file, which contains additional metadata and conforms to
-the measures schema, run:
+the measures schema, do the following:
+
+1. download the qpp quality pdfs 
+	```
+	wget https://qpp.cms.gov/docs/QPP_quality_measure_specifications.zip .
```
Comment (sungchoi) [inline README.md]: For those without wget (I think that's a linux thing, right?)
`curl -o ./QPP_quality_measure_specifications.zip https://qpp.cms.gov/docs/QPP_quality_measure_specifications.zip`

File: README.md [hunk 2]
```diff
@@ -44,9 +44,20 @@ var benchmarksSchema = qppMeasuresData.getBenchmarksSchema();
...
+	unzip QPP_quality_measure_specifications.zip
+	unzip Claims-Registry-Measures.zip
+	```
+2. run the convert from pdfs tool to get the quality info from the pdfs
+	```
+	node scripts/get-quality-measures-from-pdfs.js Claims-Registry-Measures
```
Comment (sungchoi) [inline README.md]: Getting this error:
```
node scripts/get-quality-measures-from-pdfs.js Claims-Registry-Measures
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: spawn pdftotext ENOENT
```

File: README.md [hunk 3]
```diff
@@ -44,9 +44,20 @@ var benchmarksSchema = qppMeasuresData.getBenchmarksSchema();
...
+	```
 ```
-$ jq -s add util/additional-measures.json <(curl -s https://qpp.cms.gov/api/v1/aci_measures | node scripts/convert-qpp-to-measures.js aci) <(curl -s https://qpp.cms.gov/api/v1/ia_measures | node scripts/convert-qpp-to-measures.js ia) <(curl -s https://qpp.cms.gov/api/v1/quality_measures | node scripts/convert-qpp-to-measures.js quality) | tee measures/measures-data.json
+$ jq -s add util/additional-measures.json <(curl -s https://qpp.cms.gov/api/v1/aci_measures | node scripts/convert-qpp-to-measures.js aci) <(curl -s https://qpp.cms.gov/api/v1/ia_measures | node scripts/convert-qpp-to-measures.js ia) <(curl -s https://qpp.cms.gov/api/v1/quality_measures | node scripts/convert-qpp-to-measures.js quality) | node scripts/merge-measures-data.js | tee measures/measures-data.json
```
Comment (sungchoi) [inline README.md]: This part seemed to work. (based on the existing files and not the one I was trying to recreate from scratch).

File: scripts/merge-measures-data.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      });
+
+      qppJson[index].strata = strataDetails;
+      qppJson[index].overallAlgorithm = performanceRateInfo.overallAlgorithm;
+    }
+  });
+
+  console.error('did not find measure details for the following', measuresNotFound);
+  return JSON.stringify(qppJson, null, 2);
+}
```
Comment (sungchoi) [inline scripts/merge-measures-data.js]: lint: new line

**Comment by sungchoi** [comment]
@jiahuang got this error when I tried to validate the measures data.
(cat measures/measures-data.json | node scripts/validate-data.js measures)
```
Invalid: data[124] should match exactly one schema in oneOf
```

**Comment by jiahuang** [comment]
oh good call, i changed the schema but did not update the validation script

File: README.md (hunk 1 above)
Comment (jiahuang) [inline README.md]: or maybe i should add a `brew install wget` call first? also need to do a `brew install jq` for the next step anyway

File: README.md (hunk 2 above)
Comment (jiahuang) [inline README.md]: Hm, I'm having trouble recreating this. Is the Claims-Registry-Measures folder unzipped in the path that you ran this from?

Comment (jiahuang) [inline README.md]: added a command to install poppler that fixes this.

File: scripts/merge-measures-data.js (hunk 4 above)
Comment (jiahuang) [inline scripts/merge-measures-data.js]: fixzed

**Comment by sungchoi** [comment]
@jiahuang LGTM

File: measures/measures-schema.yaml [hunk 5]
```diff
@@ -99,12 +99,11 @@ definitions:
         default: false
       overallAlgorithm:
         description: Formula to determine the overall performance rate, given multiple strata of performance rates.
-        enum: [simpleAverage, weightedAverage, first, last]
+        enum: [simpleAverage, weightedAverage, numerators]
```
Comment (marimiyachi) [inline measures/measures-schema.yaml]: What are numerators? Could you update the description maybe to describe?

File: util/quality-measures-additional-info.json [hunk 6]
```diff
@@ -0,0 +1,1217 @@
+[
+  {
```
Comment (marimiyachi) [inline util/quality-measures-additional-info.json]: Do the descriptions need to be in a separate file? Also maybe the file name could be more explicit `quality-measures-strata` or something?

File: measures/measures-schema.yaml (hunk 5 above)
Comment (jiahuang) [inline measures/measures-schema.yaml]: numerators is just adding up only the numerators. I'll update the description.

File: util/quality-measures-additional-info.json (hunk 6 above)
Comment (jiahuang) [inline util/quality-measures-additional-info.json]: The descriptions are programmatically generated from the pdfs, but the shortened names (and things like the algorithm used to calculate them) are manually changed. 👍 on a more descriptive name.

Comment (marimiyachi) [inline util/quality-measures-additional-info.json]: Ok - it's worth adding a note that there is manual intervention involved in the README.

File: README.md [hunk 7]
```diff
@@ -44,9 +44,20 @@ var benchmarksSchema = qppMeasuresData.getBenchmarksSchema();
 
 ### Generating measures-data.json
 To regenerate the `measures-data.json` file, which contains additional metadata and conforms to
-the measures schema, run:
+the measures schema, do the following:
+
+1. download the qpp quality pdfs
```
Comment (marimiyachi) [inline README.md]: nit: uppercase here and after 2)

**Pull request closed without merging**